pytest_catchlog.py

```diff
@@ -139,10 +180,13 @@ def emit(self, record):
 class CatchLogFuncArg(object):
```

Comment:
"FuncArg" is basically the old concept of fixtures - see [reasoning for fixture/funcarg evolution](http://pytest.org/latest/funcarg_compare.html) - so I don't think this should be called funcarg anymore now that we use fixtures instead of `pytest_funcarg__capturelog` (also see the docstring of `__init__`).